.github/workflows/check-lib.yml

```
@@ -60,9 +60,9 @@ jobs:
 
       - name: Build docs
         env:
-          RUSTDOCFLAGS: --cfg docsrs -D broken_intra_doc_links
+          RUSTDOCFLAGS: --cfg docsrs
```

This flag has been renamed, there is no reason to remove it completely.
```suggestion
          RUSTDOCFLAGS: --cfg docsrs -D rustdoc::broken_intra_doc_links
```

---

It's already enabled in all crates (`lib.rs`) so I don't see why it should be passed two times